
Raw Benchmark Output
====================

Running Lattice Boltzmann D2Q9 on Octuplets using Bohrium/CPU
    commit: `#ac496192fc2431d5fdf7635eccf58f8bdc115379 <https://bitbucket.org/bohrium/bohrium/commits/ac496192fc2431d5fdf7635eccf58f8bdc115379>`_,
    time: 2015-03-13 04:06:01.254858.

    command: ``python benchmark/Python/lattice_boltzmann_D2Q9.py --size=1000*1000*10 --bohrium=True``

Run 00
~~~~~~
    stdout::

        N/A

    stderr::

        python: can't open file 'benchmark/Python/lattice_boltzmann_D2Q9.py': [Errno 2] No such file or directory
        



Run 01
~~~~~~
    stdout::

        N/A

    stderr::

        python: can't open file 'benchmark/Python/lattice_boltzmann_D2Q9.py': [Errno 2] No such file or directory
        



Run 02
~~~~~~
    stdout::

        N/A

    stderr::

        python: can't open file 'benchmark/Python/lattice_boltzmann_D2Q9.py': [Errno 2] No such file or directory
        



